surfacers/postgres/proto/config.proto

```diff
@@ -7,4 +7,5 @@ option go_package = "github.com/google/cloudprober/surfacers/postgres/proto";
 message SurfacerConf {
   required string connection_string = 1;
   required string metrics_table_name = 2;
+  optional int64 max_concurrent_inserts = 3 [default = 1000];
```

Comment:
Can you please name it metrics_buffer_size and set default to 10000. Impact of larger buffer size is pretty small. These changes will make it consistent with prometheus surfacer:
https://github.com/google/cloudprober/blob/037977dbc1716b367529d276c471571d39690dfc/surfacers/prometheus/proto/config.proto#L13

Reply:
Done